testlogger/testlogger2.cpp

```diff
@@ -4,10 +4,10 @@
 #include <QTimer>
 #include <QDateTime>
 
-#define TEST_FILE_ROTATION 1
+#define TEST_FILE_ROTATION 0

```

Comment:
I'm wondering the reason of these defines.
Why do not have different tests for those options, so to keep all of them under test?

Reply:
this test belongs to a talk given at qtdays so the defines were used to show a specific feature

Reply:
Consider to file a new issue in order to run all tests w/o the need to change the defines.